libs/langchain-openai/src/azure/chat_models.ts

```diff
@@ -425,12 +427,27 @@ import {
  * </details>
  */
 export class AzureChatOpenAI extends ChatOpenAI {
+  azureOpenAIApiVersion?: string;
+
+  azureOpenAIApiKey?: string;
+
+  azureADTokenProvider?: () => Promise<string>;
+
+  azureOpenAIApiInstanceName?: string;
+
+  azureOpenAIApiDeploymentName?: string;
+
+  azureOpenAIBasePath?: string;
+
+  azureOpenAIEndpoint?: string;
```

Comment:
do these match python / should we take this as an opportunity to make the variable names match, or keep as is to make as little breaking as possible?

Reply:
Yeah I think too big a break

Reply:
I don't even understand what any of these do